### DISCUSSION TITLE
Jove can be killed without destroying 3 pylons

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15159] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Currently jove has 100% damage + burn resistance when it's above 70% health
and in vanilla we only have doomofjove affliction applied when pylon destroyed that can damage jove
normally this won't cause any issue unless other mod introduces new health decreasing afflictions
but now we have Europabrew's effects which can at most increase 100% damage taken
This allows you to damage jove without destroying any pylon

one solution is making jove Unkillable until 3 pylons are destroyed (make pylon trigger a scripted event when destroyed and use CountTargetsAction to check if all 3 are broken)

### Reproduction steps

1. use Europabrew on jove
2. try to damage it

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by NotWendy** [comment]
Sounds like a bug which legitimately makes for a more pleasant experience, considering just how badly designed the fight is.
(Please fix the bug anyways.)
